src/app/models/harvest/HarvestItem.ts

```diff
@@ -0,0 +1,70 @@
+import { Injector } from "@angular/core";
+import { AUDIO_RECORDING, SHALLOW_HARVEST } from "@baw-api/ServiceTokens";
+import { DateTimeTimezone, HasUpdater, Id } from "@interfaces/apiInterfaces";
+import { bawDateTime } from "@models/AttributeDecorators";
+import { AudioRecording } from "@models/AudioRecording";
+import { AbstractModel } from "../AbstractModel";
+import { hasOne, updater } from "../AssociationDecorators";
+import { User } from "../User";
+import { Harvest } from "./Harvest";
+import { HarvestItemInfo, IHarvestItemInfo } from "./HarvestItemInfo";
+
+/**
+ * State of a harvest item
+ *
+ * @param new File on disk found
+ * @param metadataGathered Analyzed the file, gotten the metadata, and validated
+ * @param failed File is not valid for some known reason
+ * @param completed Successfully harvested the file
+ * @param errored An error occurred while harvesting the file
+ */
+export type HarvestItemState =
+  | "new"
+  | "metadataGathered"
+  | "failed"
+  | "completed"
+  | "errored";
+
+export interface IHarvestItem extends HasUpdater {
+  id?: Id;
+  harvestId?: Id;
+  audioRecordingId?: Id;
+  createdAt?: DateTimeTimezone | string;
+  deleted?: boolean;
+  info?: HarvestItemInfo | IHarvestItemInfo;
+  path?: string;
+  status?: HarvestItemState;
```

Comment:
So I'm thinking about omitting `info`.

I'll probably just return the `validations` sub-object from `info` and not the rest of `info`.